lms/models/lti_params.py

```diff
@@ -138,8 +138,28 @@ def _to_lti_v11(v13_params):
         v11_params.update({f"custom_{key}": value for key, value in custom.items()})
 
     if "roles" in v11_params:
-        # We need to squish together the roles for v1.1
-        v11_params["roles"] = ",".join(v11_params["roles"])
+        context_roles = []
+        for role in v11_params["roles"]:
```

Comment:
I think this might work nicely as a list comprehension along the lines of:

```python
role for role in v11_params["roles"]
if "http://purl.imsglobal.org/vocab/lis/v2" not in role
or role.startswith("http://purl.imsglobal.org/vocab/lis/v2/membership")
```

Reply:
I had it originally as this but when adding the comments I found it more difficult to read.